db/migrate/20200319165536_add_job_role_to_vacancies.rb

```diff
@@ -0,0 +1,5 @@
+class AddJobRoleToVacancies < ActiveRecord::Migration[5.2]
+  def change
+    add_column :vacancies, :job_role, :string, array: true
```

Comment:
This should be called `job_roles` everywhere, it it hasn't been deployed anywhere, I would change it before getting it merged